### DISCUSSION TITLE
Alternatives to undetected chromedriver and SeleniumBase?

### DISCUSSION THREAD

**Discussion by MacMarde** [ultrafunkamsterdam/undetected-chromedriver#1677] [General]
A while ago the UC was working very well.

But it stopped working, because the anti-bot detection industry found a way to detect it again.

Unti now only SeleniumBase offers an alternative afaik, but the solutions is not very bullet proof.

Maybe we are searching on the wrong end.

The question is are there other ways to make a bot undetectable? For example using Firefox and Selenium or even completely other solutions like puppeteer and or other programming languages / browsers /services etc. Anything goes.

There are lots of possiblities out there. So you are welcome to share anything that might help.

**Comment by justinearl** [comment]
You may check this https://github.com/ulixee/hero
